### ISSUE TITLE
VoiceOver does not announce message reactions

### ISSUE THREAD

**Issue by ogomez92** [deltachat/deltachat-ios#2577]
- iOS version: 18.3
- Device: iPhone 15 pro 
- Delta Chat version: 1.50.5
- Expected behavior: When a chat is open and VoiceOver announces a message, reactions should be announced i.e. John, Joe and Jill reacted with thumbsup
- Actual behavior: Reacitons are skipped and not announced
- Steps to reproduce the problem: Hover over any chat via voiceover focus/flicks which has reactions
- Screenshots: I can post a video but I think it's clear, let me know if you need.
- Logs: UI issue

**Comment by r10s** [comment]
note for other testers when using voiceover:
- Return to the Home screen: From the bottom of the screen slide one finger up until the second click. Lift your finger from the screen.
